.github/workflows/install.yml

```
@@ -225,17 +261,38 @@ jobs:
           CHUNKS="$(./scripts/ci/generate-docker-chunks.sh docker-express docker-next docker-static docker-playwright docker-jest)"
           echo "CHUNKS: '$CHUNKS'"
           if [[ "$CHUNKS" != "[]" ]]; then
             echo BUILD_CHUNKS=$(echo "$CHUNKS" | jq -cM '. | map("\(.|tostring)")') >> "$GITHUB_OUTPUT"
-            echo "CHUNKS='$CHUNKS'" >> "$GITHUB_OUTPUT"
           fi
+          echo "CHUNKS='$CHUNKS'" >> "$GITHUB_OUTPUT"
+
+      - name: Prepare deployment targets
+        id: deploy_map
+        if: ${{ steps.build_projects.outputs.CHUNKS != '[]' }}
+        run: |
```

actionlint (1.7.4)

property "build_map" is not defined in object type {build_projects: {conclusion: string; outcome: string; outputs: {string => string}}; docker_projects: {conclusion: string; outcome: string; outputs: {string => string}}; docker_tags: {conclusion: string; outcome: string; outputs: {string => string}}; e2e_projects: {conclusion: string; outcome: string; outputs: {string => string}}; export-sha: {conclusion: string; outcome: string; outputs: {string => string}}; get-branch: {conclusion: string; outcome: string; outputs: {string => string}}; git-bot-config: {conclusion: string; outcome: string; outputs: {string => string}}; git_nx_base: {conclusion: string; outcome: string; outputs: {}}; lint_projects: {conclusion: string; outcome: string; outputs: {string => string}}; prepare-merge-queue: {conclusion: string; outcome: string; outputs: {string => string}}; set-outputs: {conclusion: string; outcome: string; outputs: {string => string}}; test_projects: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)